pkgs/development/libraries/luabind/default.nix

```diff
@@ -1,25 +1,25 @@
-{ lib, stdenv, fetchFromGitHub, boost-build, lua, boost }:
+{ lib, stdenv, fetchFromGitHub, boost-build, lua, boost, cmake }:
 
 stdenv.mkDerivation rec {
   pname = "luabind";
   version = "0.9.1";
 
   src = fetchFromGitHub {
-    owner = "luabind";
+    owner = "Oberon00";
     repo = "luabind";
-    rev = "v${version}";
-    sha256 = "sha256-sK1ca2Oj9yXdmxyXeDO3k8YZ1g+HxIXLhvdTWdPDdag=";
+    rev = "49814f6b47ed99e273edc5198a6ebd7fa19e813a";
+    sha256 = "sha256-JcOsoQHRvdzF2rsZBW6egOwIy7+7C4wy0LiYmbV590Q";
   };
 
-  patches = [ ./0.9.1_modern_boost_fix.patch ./0.9.1_boost_1.57_fix.patch ./0.9.1_discover_luajit.patch ];
+  # patches = [ ./0.9.1_modern_boost_fix.patch ./0.9.1_boost_1.57_fix.patch ./0.9.1_discover_luajit.patch ./0.9.1_darwin_dylib.patch ];
```

Comment:
The unused patches can be deleted?

And any reason to keep the commented out code (`# patches = ...`) here?